vcluster/_fragments/sleepmode-istio-example.mdx

```diff
@@ -0,0 +1,116 @@
+import Flow, { Step } from '@site/src/components/Flow';
+import CodeBlock from '@theme/CodeBlock'
+import SleepmodeIstioConfig from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/_code/sleepmode-istio-config.yaml'
+import SleepmodeIstioExample from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/experimental/_code/sleepmode-istio-examples.yaml'
+
+### Configure sleep mode for use with Istio
+
+<Flow id="sleepmode-ingress-example">
+  <Step>
+Create the `kind` cluster.
+
+```shell title="Create a kind cluster"
+kind create cluster --name istio-demo --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    extraPortMappings:
+      - containerPort: 30000
+        listenAddress: 127.0.0.1
+        hostPort: 80
+        protocol: TCP
+      - containerPort: 31000
+        listenAddress: 127.0.0.1
+        hostPort: 443
+        protocol: TCP
+      - containerPort: 32000
```

Comment:
Where do these ports come from, are they random just for this setup ?

Reply:
Random(ish). The Istio install will choose random container ports.  Setting them explicitly here lets us patch the istio service when it starts to be able to hit port 80 from the host with curl. I can add an explanation.

Reply:
Ok, can you just clarify that in this. Like: "Istio will have specific ports, set them here..."